F1WM/Utilities/MappingProfile.cs

```diff
@@ -59,6 +59,12 @@ public MappingProfile()
 				.ForMember(api => api.FinishPosition, o => o.MapFrom(db => db.StartPosition))
 				.ForMember(api => api.Status, o => o.MapFrom(db => db.StartStatus.GetQualifyStatus()))
 				.AfterMap((db, api) => db.FillSessionsInfo(api));
-		}
+            CreateMap<Race, CalendarRace>()
+                .ForMember(api => api.translatedName, o => o.MapFrom(db => db.Country.GenitiveName.GetGrandPrixName()))
+                .ForMember(api => api.raceid, o => o.MapFrom(a => a.Id))
```

Comment:
You could have saved this line if you called CalendarRace.raceid CalendarRace.Id instead